blocks/pipeline.py

```diff
@@ -51,12 +55,11 @@ def is_fitted(self) -> bool:
         return all([model.is_fitted() for model in self.models])
 
     def save(self) -> None:
-        for model in self.models:
-            model.save(self.id)
+        pass
```

Comment:
hmm, we don't need this anymore? how do we propagate it?

Reply:
I think train_predict loops through all models so it is already propagated there.

Reply:
We should keep it as a pass function otherwise the entire pipeline will get saved as a python object. To be honest we should discuss if its a good default behaviour or not.

Reply:
hmm yes, maybe that's not ideal. maybe we can create a PickleIOMixin that does that explicitly (overriding .save() and .load() methods), so we can explicitly import it where needed?
For example, we may not want to do this for many augmenters, etc - it may be a discrete decision we'd need to make? not sure.